demos/signal-protocol/python/env_server.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+import saw_client as saw
+import saw_client.connection as conn
+
+
+def env_connect():
```

Comment:
Does the `saw_client` library also implement something like this now? Is it necessary anymore? And, if yes, should it be moved?

Reply:
Ah, good catch. Now that I check, `saw_client` now properly automatically if `saw-remote-api` is on your `PATH` (which is something we already assume in `saw-demos`) and uses that behind the scenes. This means I can get rid of this module entirely.